### DISCUSSION TITLE
各类提醒的配置逻辑和优先级

### DISCUSSION THREAD

**Discussion by meiyoushuru** [ClassIsland/ClassIsland#59] [Q&A] [answered]
在时间表等各个选项中均可配置上下课提醒，请问这些提醒的配置逻辑和优先级是怎样的呢？或者说，我应该如何配置提醒？

**Comment by HelloWRC** [comment, marked as answer]
这个问题在应用的帮助文档的`进阶功能`文章有解释。这些设置会以科目、时间点、课表、时间表的顺序应用。如果这些地方都没有进行配置，那么就使用在应用设置设置的默认设置。
